ntcore/generate_topics.py

```diff
@@ -13,28 +14,36 @@ def Output(outPath, outfn, contents):
 
     outpathname = f"{outPath}/{outfn}"
 
-    if os.path.exists(outpathname):
-        with open(outpathname, "r") as f:
-            if f.read() == contents:
-                return
-
```

Comment:
Why have it? I imagine the reason to have this was to not pollute the gradle / cmake cache because the task would be run for every invocation. With the check being done asynchronously the fact that the file exists shouldn't matter. 

This was necessary for me before when the files were generated during on build, if I said this file would be generated and it wasn't, bazel would get angry.